### ISSUE TITLE
Data Analysis with Python Misleading Quiz Question

### ISSUE THREAD

**Issue by jeremylt** [freeCodeCamp/freeCodeCamp#42343]
**Affected page**
https://www.freecodecamp.org/learn/data-analysis-with-python/data-analysis-with-python-course/introduction-to-data-analysis

**Issue**
This question implies that R is better for data science than Python. This is a pretty debatable point, and not really a question to gauge the learner's level of understanding.

> Why should you choose R over Python for data analysis?

We can mitigate this issue somewhat by changing the text of the question to something more neutral, such as

> Why might you choose to use R over Python for data analysis?

Though, really this seems like a poor question to summarize the contents of the slide deck/presentation.

I think we should either update the question to more neutral language or (preferably) pick a better question to summarize the important portions of this lesson (language wars are not 'important' content for us to quiz users on, IMHO).

---

Speaking of the slides,

They say:

From the slides for this lesson:
> Python, sadly, is not always the answer

> * When R Studio is needed
> * When dealing with advanced statistical methods
> * When extreme performance is needed

It is a dubious claim that R is more performant than Python. The performance difference between the two depends upon the problem at hand, the skill of the programmer, the programming paradigmns used, etc. For some situations, Python can be much faster than R.

With the explosion of AI/ML/Data Science packages in Python in recent years, it is also somewhat dubious that R is better for 'advanced statistical methods', as the range of analysis packages for Python has grown a considerable amount.

Each language has problems they are more commonly used for, but I wouldn't really say one is 'better' in the way this content is implying. The situation is much more grey than the black and white picture the slides paint.

I think this sort of stale content is a risk with our video based lessons, where it is harder to update content as technology changes.

**Comment by MJ94** [comment]
Thanks for opening this issue, @jeremylt. For reference, the forum topic from which this originated is called [Slightly Confusing “Introduction to Data Analysis” Quiz Question](https://forum.freecodecamp.org/t/slightly-confusing-introduction-to-data-analysis-quiz-question/463110).

**Comment by jeremylt** [comment]
I knew I was forgetting something when I added the issue! Thanks for adding that.

**Comment by raisedadead** [comment]
>  pick a better question to summarize the important portions of this lesson

I agree. We should proceed with this. Open to recommendations before opening this up for contributions, or a PR if you are interested.

You are correct, we will be phasing out video-based challenges in favour of interactive curriculum because they are really hard to maintain and fix in case of inaccuracieses.

**Comment by jeremylt** [comment]
I have seen this message, but I'm suddenly juggling several large plates, so I have not had time to circle back.

**Comment by raisedadead** [comment]
No worries @jeremylt, we understand you are busy. Thanks for helping us with so many things :) 

If you recommend some changes we can tag this as help wanted. I am not sure if changing the question to `Why might you choose to use R over Python for data analysis?` solves the concern adequately. 

I am fine if you think that is enough though.

**Comment by raisedadead** [comment]
Hi @aknouche we haven't reached a conclusion. You should take a look at the help wanted issues that are open for contributions at this time.

**Comment by MJ94** [comment]
As the original requester, I support the question rewritten as suggested at the very least. However, I don't think the question as rewritten captures the essence and key takeaways of the video. We don't need to make it a "fight" between R and Python.

**Comment by jeremylt** [comment]
I was thinking of something along the lines of 

Which of the following is not part of Data Analysis

1) Building statistical models and data visualizations

2) Fixing incorrect values and removing invalid data

3) Picking a desired conclusion for the analysis <--- THIS IS THE BIG NO-NO

4) Transforming data into an appropriate data structure

---


By the way, the slide deck seems to include an advertisement. Just wanted to make sure that was intentional:
![image](https://user-images.githubusercontent.com/25011573/121228106-956c4280-c849-11eb-9ecf-fd456c852824.png)

---

Also, there is a typo in the emoji for "Extremely Powerful", suggesting this is a downside
![image](https://user-images.githubusercontent.com/25011573/121228205-b5036b00-c849-11eb-9fa1-70b550bb8f5b.png)

**Comment by beaucarnes** [comment]
@jeremylt Thanks for pointing out the advertisement. That is not supposed to be there.

The video and slides in this section of the curriculum is came from a part of a video on the freeCodeCamp YouTube channel. I did not realized that the slides that were linked with the video contained the ad. I'm creating a new version of the slides as we cannot edit the original version. I'm going to update the link in the curriculum with the new slides that do not contain the advertisement.

I'll also update that emoji typo. However, the emoji typo is also in the video and we'll just have to leave that in.

Also, I agree with Jeremy's suggested quiz question. Since I'll be updating the slides link, I'll update the question as well.
